material-overrides/main.html

```diff
@@ -193,5 +193,65 @@
         }
       }
     });
-  </script>  
+  </script>
+  <script>
+    document.addEventListener('DOMContentLoaded', () => {
```

Comment:
You've added part of this script in the clipboardCopy.js file in your docs PR: https://github.com/polkadot-developers/polkadot-docs/pull/786/files#diff-a37cfed21320b0fc4199b9ea0c3918d5f0fc2dc50d14a6fb86ecf20657c5db8dR1